src/lib/middleware/bm_network.cpp

```diff
@@ -28,15 +28,21 @@ static constexpr uint8_t networkTopicType = 1;
  * \param type Network type to send over. MUST be BM_NETWORK_TYPE_CELLULAR_IRI_FALLBACK.
  * \return True if the data was successfully queued for transmission.
  */
-bool spotter_tx_data(const void* data, uint16_t data_len, bm_serial_network_type_e type) {
-    bool rval = false;
-    size_t msg_len = sizeof(bm_serial_network_data_header_t) + data_len;
-    uint8_t * data_buf = static_cast<uint8_t*>(pvPortMalloc(msg_len));
-    configASSERT(data_buf);
-    bm_serial_network_data_header_t * header = reinterpret_cast<bm_serial_network_data_header_t *>(data_buf);
+bool spotter_tx_data(const void *data, uint16_t data_len, bm_serial_network_type_e type) {
+  bool rval = false;
+  size_t msg_len = sizeof(bm_serial_network_data_header_t) + data_len;
+  uint8_t *data_buf = static_cast<uint8_t *>(pvPortMalloc(msg_len));
+  configASSERT(data_buf);
+  do {
+    if (data_len > SPOTTER_TX_MAX_TRANSMIT_SIZE_BYTES) {
```

Comment:
Small change for size checking

Reply:
NICE! 👏 Thank you! This will allow Bristlemouth users to check the return value to know about a failure that was invisible to them before.